Library/Homebrew/dev-cmd/formula.rb

```diff
@@ -1,32 +1,34 @@
-# typed: true
+# typed: strict
 # frozen_string_literal: true
 
+require "abstract_command"
 require "formula"
 require "cli/parser"
 
 module Homebrew
-  module_function
+  module DevCmd
+    class FormulaCmd < AbstractCommand
+      sig { returns(String) }
+      def self.command_name = "formula"
```

Comment:
For this command, as well as `livecheck`, naming the class in the usual manner (i.e. `Formula` and `Livecheck`) causes a bunch of type resolution errors in the command directories. Rather than resolve those to use fully-qualified paths, I gave the classes different names, and overrode their `command_name` values so that the command still works in the usual way.

Reply:
I wonder if it's worth automatically stripping `-cmd` suffixes from the generated `command_name` so we don't need to do this every time. (And we can use `self.command_name=` in case we actually want to keep `-cmd`).